### ISSUE TITLE
("Couldn't retrieve t field from login response", <Response [200]>)

### ISSUE THREAD

**Issue by a97077088** [Terrance/SkPy#139]
("Couldn't retrieve t field from login response", <Response [200]>)

is api update

**Comment by Terrance** [comment]
Discussion in #65.

**Comment by hmcvieira** [comment]
it is not a duplicate

**Comment by OskJonsdottir** [comment]
Same problem here, the library has worked for me without trouble for 5 months uninterruptedly, never changing the IP address and barely changing the usage rate. Suddenly, it stopped working today. I've tried logging using web Skype from the same machine, to no avail. I have created a new Skype account and tried logging in again, and it failed at the first attempt. It looks to me like an API change.

**Comment by OskJonsdottir** [comment]
Same problem here, the library has worked for me without trouble for 5 months uninterruptedly, never changing the IP address and barely changing the usage rate. Suddenly, it stopped working today. I've tried logging using web Skype from the same machine, to no avail. I have created a new Skype account and tried logging in again, and it failed at the first attempt. It looks to me like an API change.

**Comment by OskJonsdottir** [comment]
Same problem here, the library has worked for me without trouble for 5 months uninterruptedly, never changing the IP address and barely changing the usage rate. Suddenly, it stopped working today. I've tried logging using web Skype from the same machine, to no avail. I have created a new Skype account and tried logging in again, and it failed at the first attempt. It looks to me like an API change.

**Comment by OskJonsdottir** [comment]
Same problem here, the library has worked for me without trouble for 5 months uninterruptedly, never changing the IP address and barely changing the usage rate. Suddenly, it stopped working today. I've tried logging using web Skype from the same machine, to no avail. I have created a new Skype account and tried logging in again, and it failed at the first attempt. It looks to me like an API change.
